src/cascadia/TerminalSettingsEditor/Profiles.xaml

```diff
@@ -78,7 +84,21 @@ the MIT License. See LICENSE in the project root for license information. -->
                         </StackPanel>
                         <StackPanel Grid.Column="0" Grid.Row="1" Margin="0,0,100,0">
                             <Controls:NumberBox x:Uid="Profile_FontSize" Margin="0,0,0,20" FontSize="15" Value="{x:Bind State.Profile.FontSize, Mode=TwoWay}" SpinButtonPlacementMode="Compact" SmallChange="1" LargeChange="10" ToolTipService.Placement="Mouse" />
-                            <TextBox x:Uid="Profile_FontWeight" Margin="0,0,0,20" FontSize="15" ToolTipService.Placement="Mouse"/>
+                            <StackPanel>
+                                <ComboBox x:Uid="Profile_FontWeight"
+                                          x:Name="FontWeightComboBox"
+                                          Margin="0,0,0,20"

```

Comment:
why is the margin on the combobox and not the stackpanel? Should it be on the stackpanel?